public/index.html

```diff
@@ -6,6 +6,8 @@
     <body>
 	<div id="container">
 	</div> <!-- container -->
+	<div id="letter">
+	  </div>
```

Comment:
super unimportant but there's just an extra space on the closing div element making it not line up with the opening div.